src/utils/ServerSocket.js

```diff
@@ -41,6 +47,14 @@ export default class ServerSocket {
         });
     }
 
+    setRtcManager(rtcManager) {
+        if (this.glitchFree) {
+            if (!this._reconnectManager) throw new Error("ReconnectManager is missing");
```

Comment:
I don't see anyway to throw this error if `glitchFree` is true. If we keep this throw then I feel like we should handle that correctly in the `RtcManagerDispatcher.js` file change also. But we should just remove it IMO:

```suggestion
        if (this._reconnectManager) {
```